### DISCUSSION TITLE
Is it possible to use multiple interpolation characters for verbatim strings

### DISCUSSION THREAD

**Discussion by fgheysels** [dotnet/csharplang#6473] [Q&A]
I have a verbatim string that represents a JSON string, like this:

```csharp
string s = @"{""property"": ""somevalue""}";
```

Now, I'd like to change `"somevalue"` to the contents of a variable using string interpolation, so I thought I'd be able to do this:

```csharp
string foo = "something else";
string s = @$$"{""property"": ""{{foo}}""}";
```` 

or even this:

```csharp
string foo = "something else";
string s = @$$"""{"property": "{{foo}}"}""";
```` 

However, the compiler issues this error:

> CS9008: Sequence of '@' character is not allowed. A verbatim string or identifier can only have one '@' character and a raw string cannot have any.

If I switch the verbatim indicator and the string-interpolation indicator, the same error remains.

Is it not possible to combine both in this case ?  
I have specified the 'Preview' LangVersion in my csproj file.

**Comment by HaloFour** [comment]
You don't use the `@` with raw strings:

```cs
string foo = "something else";
string s = $$"""{"property": "{{foo}}"}""";
```

IIRC the ability to double-up interpolation indicators is not supported with verbatim strings, only raw strings.

**Comment by FaustVX** [comment]
You have raw string literals: #4304
[sharplab.io](https://sharplab.io/#v2:EYLgtghglgdgNAFxAJwK7wCYgNQB8ACATAAwCwAUEQIwUX5UCcAFACQsBEnA3uwBICmAG0EB7diAAE7AAqiADuwC+ndgEoA3EA==)

Ok, @HaloFour answered just before :)
